editor/plugins/animation_blend_space_2d_editor.cpp

```diff
@@ -818,11 +818,9 @@ void AnimationNodeBlendSpace2DEditor::_notification(int p_what) {
 
 			String error;
 
-			if (!tree->is_active()) {
-				error = TTR("AnimationTree is inactive.\nActivate to enable playback, check node warnings if activation fails.");
-			} else if (tree->is_state_invalid()) {
-				error = tree->get_invalid_state_reason();
-			} else if (blend_space->get_triangle_count() == 0) {
+			error = tree->get_editor_error_message();
+
+			if (blend_space->get_triangle_count() == 0) {
```

Comment:
To preserver the previous behavior, you should also check whether `error` is empty here.